rust/processor/src/models/user_transactions_models/signatures.rs

```diff
@@ -88,8 +96,21 @@ impl Signature {
             SignatureEnum::MultiEd25519(_) => String::from("multi_ed25519_signature"),
             SignatureEnum::MultiAgent(_) => String::from("multi_agent_signature"),
             SignatureEnum::FeePayer(_) => String::from("fee_payer_signature"),
-            SignatureEnum::SingleSender(_sender) => {
-                panic!("SingleSender signature is not supported")
+            SignatureEnum::SingleSender(sender) => {
```

Comment:
This sort of looks like code dupe from above, is there a way to pull this out into a common function? 

At the least we should probs use constants for the strings.